src/SourceBuild/content/eng/pipelines/ci.yml

```diff
@@ -1,31 +1,27 @@
-# This is the main build definition (PR+CI) for dotnet/dotnet
+# This is the main build definition (CI) for dotnet/dotnet
+# This pipeline runs on a schedule every weekday at midnight (00:00 UTC) for "main" and when changes are pushed to "release/" and "internal/release/".
+
+schedules:
+- cron: '0 0 * * Mon-Fri'
+  displayName: Weekday midnight build
+  branches:
+    include:
+    - main
+  batch: true
 
 trigger:
   batch: true
   branches:
     include:
-    - main
     - release/*
     - internal/release/*
     exclude:
     - release/*.0.2xx

```

Comment:
These exclusions were added in https://github.com/dotnet/installer/pull/17578 in response to https://github.com/dotnet/installer/pull/17558#issuecomment-1768444370. Could removing the exclusions again allow the issue discussed in  https://github.com/dotnet/installer/pull/17578 to persist?

Reply:
@mthalman - Can you answer this?  Are the trigger exclusions read from main or the target branch?  If they aren't read from the target branch, this change is going to mess up the release branch CI in general.

Reply:
I assumed they were from the main branch but I'm not sure anymore after encountering that other PR that targeted 8.0.2xx and it was continually trying to run the pipeline until we excluded it in that branch.

Reply:
After a little research, it appears that the intended behavior is for the triggers to be read from the main branch. With this in mind, and given that in 9.0 we intend to support all feature branches,  I removed the trigger exclusions in https://github.com/dotnet/installer/pull/17370/commits/4a2d4570f11911908efa9865e7e208183c709547.

Since we are still unsure of why that other PR that targetted 8.0.2xx was trying to run the pipeline, I am open to adding the exclusions back in until 9.0.

Reply:
@MichaelSimons @mthalman I tested the pipeline triggers with a test AzDo organization, and the triggers function as expected. Thus, I'm still unsure of why that other PR was attempting to run the main pipeline. 

Since this leaves us at a bit of a crossroad, I can either add in the excluded branches to err on the side of caution or not exclude these branches. What are your thoughts?

Reply:
I think we should leave the excluded branches out.